CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing
+
+## Pull Request Guidelines
+
+External contributions to Alt-Ergo may be proposed using git's standard pull request mechanism.
+The following terms apply to all such contributions:
+
+* We require all pull requests to pass :
+
+	* Our non regression suite, you can run `make non-regression` to check if your changes affect the performance and answers of Alt-Ergo.
+
+	* Our style checking, every lines should length 80 columns maximum and should be indent with ocp-indent.
+	  We strongly recommand to use the following command line so the indent and style checking will be done automatically when commiting.
+
+	```cp rsc/extra/pre-commit-git-hook .git/hooks/pre-commit```
+
+* Pull request should be rebased and up to date from the `base` branch to be merged.
+
+* All commits will be squashed into one when merged into the `base` branch. A PR with only one commit is appreciated since it facilitates the review.
```

Comment:
I don't really agree with this comment. Some changes are way way way better reviewed if commits are clean and each do a specified task. So while specifying the commits will be squashed when the PR is merged is fine, I wouldn't recommand PR to come as single commits. Rather Prs should come with something like a clean commit history, with each commit properly named and annotated with a message.

Reply:
Same as above, thanks